src/components/pages/OERInformation.tsx

```diff
@@ -0,0 +1,135 @@
+'use client';
```

Comment:
Do you need the boundary that high ? Would it be ok to lower it ?

Reply:
would be indeed cool! But we need to use https://github.com/i18next/next-i18next
I move this to another issue.